<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>494</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>494</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>494</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/image/_displayLogo.png</normaloff>:/image/_displayLogo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/image/_loginPage.jpg);</string>
  </property>
  <widget class="QLabel" name="label_Topic">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>600</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;楷体&quot;;
background: transparent;
color: white;</string>
   </property>
   <property name="text">
    <string>          齿轮故障管理系统</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Password">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border:2px solid #7b7b7b;
	border-radius:8px;
	background: rgb(170, 170, 255);
}
</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Password">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20 14pt &quot;楷体&quot;;
background: transparent;
color: white;</string>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_UserID">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20 14pt &quot;楷体&quot;;
background: transparent;
color: white;</string>
   </property>
   <property name="text">
    <string>账号:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;background: rgb(170, 170, 255);border:1px groove gray;border-radius:4px;font: 20 14pt &quot;楷体&quot;;padding:1px 4px;border-style: outset;}
QPushButton:hover{background-color:
rgb(170, 85, 255); color: white;}
QPushButton:pressed{background-color:
rgb(170, 85, 255);border-style: inset;

</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_UserID">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border:2px solid #7b7b7b;
	border-radius:8px;
	background: rgb(170, 170, 255);
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Clear">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;background: rgb(170, 170, 255);border:1px groove gray;border-radius:4px;font: 20 14pt &quot;楷体&quot;;padding:1px 4px;border-style: outset;}
QPushButton:hover{background-color:
rgb(170, 85, 255); color: white;}
QPushButton:pressed{background-color:
rgb(170, 85, 255);border-style: inset;

</string>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Tips">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>李旭科漫画体v1.0</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: white;</string>
   </property>
   <property name="text">
    <string>基于Qt 5.14.2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Copyright">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正舒体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
